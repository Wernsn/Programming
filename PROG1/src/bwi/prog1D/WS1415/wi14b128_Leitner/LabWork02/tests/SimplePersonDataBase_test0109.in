2
c
y
2015
1
1
2
b
z
2014
1
1
2
a
x
2016
1
1
2
ca
ya
2015
1
2
2
ba
za
2014
2
1
2
aa
xa
2016
2
2
2
caa
yaa
2015
1
3
2
baa
zaa
2014
3
1
2
aaa
xaa
2016
3
3
2
caaa
yaaa
2015
1
4
2
baaa
zaaa
2014
4
1
2
aaaa
xaaa
2016
4
4
2
caaaa
yaaaa
2015
1
5
2
baaaa
zaaaa
2014
5
1
2
aaaaa
xaaaa
2016
5
5
2
caaaa
yaaaa
2015
1
6
2
baaaaa
zaaaaa
2014
6
1
2
aaaaaa
xaaaaa
2016
6
6
2
caaaaaa
yaaaaaa
2015
1
7
2
baaaaaa
zaaaaaa
2014
7
1
1
2
0




