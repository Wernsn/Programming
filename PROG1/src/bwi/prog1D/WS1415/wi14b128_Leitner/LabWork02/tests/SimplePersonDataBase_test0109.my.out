Welcome to the SimplePersonDatabase.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-28): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-29): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-30): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-30): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-30): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-30): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: Firstname: Lastname: enter year (1582-2199): enter month (1-12): enter day (1-31): 
available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: 
ID  Lastname             Firstname            Birth
--------------------------------------------------------------------
001 y                    c                    Thursday, 01.01.2015
002 z                    b                    Wednesday, 01.01.2014
003 x                    a                    Friday, 01.01.2016
004 ya                   ca                   Friday, 02.01.2015
005 za                   ba                   Saturday, 01.02.2014
006 xa                   aa                   Tuesday, 02.02.2016
007 yaa                  caa                  Saturday, 03.01.2015
008 zaa                  baa                  Saturday, 01.03.2014
009 xaa                  aaa                  Thursday, 03.03.2016
010 yaaa                 caaa                 Sunday, 04.01.2015
011 zaaa                 baaa                 Tuesday, 01.04.2014
012 xaaa                 aaaa                 Monday, 04.04.2016
013 yaaaa                caaaa                Monday, 05.01.2015
014 zaaaa                baaaa                Thursday, 01.05.2014
015 xaaaa                aaaaa                Thursday, 05.05.2016
016 yaaaa                caaaa                Tuesday, 06.01.2015
017 zaaaaa               baaaaa               Sunday, 01.06.2014
018 xaaaaa               aaaaaa               Monday, 06.06.2016
019 yaaaaaa              caaaaaa              Wednesday, 07.01.2015
020 zaaaaaa              baaaaaa              Tuesday, 01.07.2014
--------------------------------------------------------------------
20 Person(s) in database, 0 free.

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: 
cannot add. database full

available options:
1) list
2) add
3) remove
4) sort
5) find
6) settings
0) quit
your choice: 
Thank you for using the SimplePersonDatabase.

